CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ include(ccache)
 # Remove soon?
 set(USE_FFMPEG ON)
 
+if(MINGW AND NOT DEFINED USE_SYSTEM_FFMPEG)
```

Comment:
SYSTEM_FFMPEG is generally not a good idea, we should probably make a custom mingw build if people really want to use it. I don't think people do use it a lot though, so a comment would suffice.

Reply:
adding the mingw static libs would be better yeah.
the bundeled msvc libs generate missing symbol errors when used with mingw, but I can change that to a comment if that's preferred

Reply:
Nah, just add a comment here saying that this is why you are using system ffmpeg in this case.